### PR TITLE
docs: part1

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
//cc @VasilyStrelyaev

### CODE REVIEW

**Comment by MargaritaLoseva** [comment]
//cc @DevExpress/testcafe-docs

File: documentation/working-with-testcafe-studio/organizing-tests.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+* Go to the **File** menu and click **Open Test Directory**.
+* On the **Welcome** page, click **Open Folder**.
+
+> If you start recording a test from the **Welcome** page, TestCafe Studio automatically creates a test directory in the *Users/Username/TestCafeStudio* folder and gives it the tested site's name. If a directory with the same name already exists, TestCafe Studio opens it and adds a new test to it.
+
+You can also organize fixtures and .js test files into *subdirectories*. To create a subdirectory, right-click a test directory in the **Explorer** panel and select **New Directory**. In the **New directory** dialog, enter the directory name and click the **Create** button.
+
+## Fixtures
+
+A fixture is a JavaScript file with `*.testcafe` extension that can contain one or more tests for a single specified target URL. Therefore, you will usually have one test fixture per website page/route.
```
Comment (VasilyStrelyaev) [inline documentation/working-with-testcafe-studio/organizing-tests.md]: Fixtures with the `*.testcafe` extension are JSON files not JavaScript

File: documentation/working-with-testcafe-studio/running-tests.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* Click the IMAGE **Run test** button on the **Test Editor**'s toolbar.
+* Click the IMAGE **Run test** button next to the test in the **Fixture Editor** or in the **Explorer** panel.
+
+**To run a .js test file**, click the **Run test** IMAGE button next to the file in the **Explorer** panel.
+
+After starting a test run, TestCafe Studio launches specified browsers and execute tests in them in parallel. The **Reports** panel displays the test's progress.
+
+**To stop a running test**, click the IMAGE button in the **Reports** panel.
+
+If the *Pause the test when it failes* option is enabled in the used run configuration, TestCafe Studio pauses the test at the moment it fails. To finish the test, click the IMAGE **Finish** button in the browser window.
```
Comment (VasilyStrelyaev) [inline documentation/working-with-testcafe-studio/running-tests.md]: > in the used run configuration

we can omit `used` here

**Comment by VasilyStrelyaev** [comment]
I also don't like `Visual tests` as opposed to `JS code tests`.
Now, in tooltips, we use `Recorded tests` vs `JS API Tests`. I agree, they are not necessarily recorded, so maybe `Studio tests` vs `JS tests` or something

**Comment by VasilyStrelyaev** [comment]
We need to discuss it and figure out a good name

File: documentation/working-with-testcafe-studio/about-test-types.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+title: About Test Types
+permalink: /documentation/working-with-testcafe-studio/about-test-types
+---
+# About Test Types
+
+TestCafe Studio allows you to create two types of tests.
+
+## Visual Tests
+
+Visual tests consists of [test actions](creating-and-recording-tests.md#test-actions) performed with a tested page (clicks, key presses, drags, etc.). In TestCafe Studio's UI and documentation, the visual tests are called simply **tests**. You create them by recording actions or by adding actions to them manually. For more information, see [Creating and Recording Tests](creating-and-recording-tests.md).
```
[the remark this answers is not in the document]
Comment (MargaritaLoseva) [reply documentation/working-with-testcafe-studio/about-test-types.md]: Good idea!

File: documentation/working-with-testcafe-studio/running-tests.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+This topic describes how to execute tests and contains the following sections.
+
+* [Run Configurations](#run-configurations)
+* [Starting and Stopping Tests](#starting-and-stopping-tests)
+* [Running Tests in Remote Browsers](#running-tests-in-remote-browsers)
+
+## Run Configurations
+
+TestCafe Studio can run tests in all popular browser: desktop, mobile and cloud, the ones with UI or headless. It determines browsers installed on the local computer and automatically creates *run configurations* for them. You can view available configurations in the **Run Configuration** list.
```
[the remark this answers is not in the document]
Comment (MargaritaLoseva) [reply documentation/working-with-testcafe-studio/running-tests.md]: Ok, I'll remove info about cloud browsers.

**Comment by MargaritaLoseva** [comment]
fpr @VasilyStrelyaev @helen-dikareva 
Images will be added later when On-Page Actions is added to the Actions panel.

File: documentation/working-with-testcafe-studio/creating-and-recording-tests.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+## Test Actions
+
+Tests consists of *test actions* to be performed with a tested page.
+
+TestCafe Studio provides a set of actions that you can use in tests. These actions are shown in the **Actions** panel on the right of the **Test Editor**. Actions are organized into a number of categories. For example, actions of the **On-Page Actions** category simulate user actions, like clicks, key presses, drags, etc., over your tested page. The actions of the **Assertions** category allow you to perform different verifications.
+
+IMAGE
+
+Each action has parameters that specify a target element and define how the action should be performed.
```
Comment (helen-dikareva) [inline documentation/working-with-testcafe-studio/creating-and-recording-tests.md]: Strictly speaking we have some actions without target element (`presskey`, `maximazeWindow` etc.)

File: documentation/working-with-testcafe-studio/organizing-tests.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+* [Tests](#tests)
+* [Coded Test Files](#coded-test-files)
+
+The image below demonstrates the **Explorer** panel that serves to manage your test files. It shows them in a tree-like structure.
+
+IMAGE
+
+## Test Directory
+
+A test directory is a file system directory used as the root of your test-related workspace. It is used to store [fixtures](#fixtures) and [.js test files](#js-test-files). You can create a test directory anywhere on your computer and then open it in TestCafe Studio in one of the following ways.
```
Comment (helen-dikareva) [inline documentation/working-with-testcafe-studio/organizing-tests.md]: Please add `.ts` too

**Comment by VasilyStrelyaev** [comment]
Aren't these files duplicated?
`documentation/working-with-testcafe-studio/creating-and-recording-tests.md`
`documentation/working-with-testcafe-studio/recording-tests.md`

**Comment by VasilyStrelyaev** [comment]
and `documentation/working-with-testcafe-studio/test-types.md` should be removed
seems like you pushed not quite successfully

File: documentation/working-with-testcafe-studio/running-tests.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+You can run an entire test directory/subdirectory, a specific fixture, an individual test or a coded test file. To run them, use the IMAGE button. This button is available in the **Test Editor**, **Fixture Editor** and **Explorer** panel.
+
+**To run the entire test directory or subdirectory**, click the **Run all tests** button next to the directory in the Explorer panel.
+
+**To run a fixture**, click the **Run all tests** IMAGE button next to the fixture in the **Explorer** panel or on the **Fixture Editor**'s toolbar.
+
+**To run an individual test**, do any of the following:
+
+* Click the IMAGE **Run test** button on the **Test Editor**'s toolbar.
```
[the remark this answers is not in the document]
Comment (MargaritaLoseva) [reply documentation/working-with-testcafe-studio/running-tests.md]: I agree.

File: documentation/working-with-testcafe-studio/recording-tests.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+* Click the IMAGE **Record a new test** button on the **Fixture Editor**'s toolbar.
+* Click the IMAGE **Start recording a test** button on the **Welcome** page.
+
+TestCafe Studio will create a new test, add it to the fixture and start recording test actions. See [During Recording](#during-recording).
+
+**To stop recording**, close the browser or click the IMAGE **Stop** button on the **Test Editor**'s toolbar.
+
+**To resume recording**, click the **Record test actions** button on the **Test Editor**'s toolbar. TestCafe will replay the previous actions and wait for new ones.
+
+### During Recording
```
[the remark this answers is not in the document]
Comment (MargaritaLoseva) [reply documentation/working-with-testcafe-studio/recording-tests.md]: I don't like Recording Process.  May be "What You Can Do During Recording"?

**Comment by MargaritaLoseva** [comment]
fpr @churkin @helen-dikareva @VasilyStrelyaev

File: documentation/working-with-testcafe-studio/running-tests.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+* [Run Configurations](#run-configurations)
+* [Starting and Stopping Tests](#starting-and-stopping-tests)
+* [Running Tests in Remote Browsers](#running-tests-in-remote-browsers)
+
+## Run Configurations
+
+TestCafe Studio can run tests in all popular browser: desktop, mobile, the ones with UI or headless. It determines browsers installed on the local computer and automatically creates *run configurations* for them. You can view available configurations in the **Run Configuration** list.
+
+To modify congirations or create a new one, use the **Run Configurations** dialog. For each configuration, you can select one or multiple browsers and specify options that affect test execution.
```
Comment (VasilyStrelyaev) [inline documentation/working-with-testcafe-studio/running-tests.md]: > congirations

File: documentation/working-with-testcafe-studio/organizing-tests.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+The **Explorer** panel will show the directory's content.
+
+> If you start recording a test from the **Welcome** page, TestCafe Studio automatically creates a test directory in the *Users/Username/TestCafeStudio* folder and gives it the tested site's name. If a directory with the same name already exists, TestCafe Studio opens it and adds a new test to it.
+
+You can also organize test files into *subdirectories*. To create a subdirectory, right-click a test directory in the **Explorer** panel and select **New Directory**. In the **New directory** dialog, enter the directory name and click the **Create** button.
+
+## Recorded Tests Files
+
+The major type of tests you create with TestCafe Studio is **recorded tests**. These tests are stored within `*.testcafe` files. TestCafe Studio automatically creates a new `*.testcafe` file when you [start recording a test](recording-tests.md#starting-and-stopping-recording) for a new tested page.
```
Comment (VasilyStrelyaev) [inline documentation/working-with-testcafe-studio/organizing-tests.md]: > The major type of tests you create with TestCafe Studio is **recorded tests**.

Maybe it's better to say

`The type of tests you record visually or compose from test actions in the **Test Editor** is called **recorded tests**.`

File: documentation/working-with-testcafe-studio/recording-tests.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+
+**To start recording** a new test, do any of the following.
+
+* Select a desired fixture in the Explorer panel and click the **Record a New Test** button.
+* Click the IMAGE **Record a new test** button on the **Fixture Editor** toolbar.
+* Click the IMAGE **Start recording a test** button on the **Welcome** page.
+
+TestCafe Studio will create a new test, add it to the fixture and start recording test actions. See [During Recording](#during-recording).
+
+You can also create an empty test and add actions to it manually. To do this, right-click a desired fixture in the **Explorer** panel and select **New Test** from the context menu. In the **Create Test** dialog, specify the test name and click **Create**. TestCafe Studio will create a test with the specified name and open it in the **Test Editor**. Next, add actions to the test from the **Actions** panel manually or click the IMAGE **Record test actions** button on the **Test Editor** toolbar and record them.
```
[the remark this answers is not in the document]
Comment (VasilyStrelyaev) [reply documentation/working-with-testcafe-studio/recording-tests.md]: Maybe we can publish the documentation even without saying it. Recording is our main workflow after all.
And later we can add a new section or topic about it.

**Comment by VasilyStrelyaev** [comment]
Please don't merge until the website is published or screenshots are ready (so that we don't have anything unpublishable in the repository)

**Comment by MargaritaLoseva** [comment]
@dirk-pieterse Please, review the created files.

**Comment by MargaritaLoseva** [comment]
closes #11

**Pull request merged**